test/module/irohad/model/converters/pb_query_factory_test.cpp

```diff
@@ -85,18 +88,42 @@ TEST(PbQueryFactoryTest, SerializeGetSignatories){
   ASSERT_EQ(iroha::hash(*res_query.value()), iroha::hash(*query));
 }
 
-TEST(PbQueryFactoryTest, get_roles){
+TEST(PbQueryFactoryTest, SerializeGetAccountAssetsTransactionsWithPager) {
+  PbQueryFactory queryFactory;
+  QueryGenerator queryGenerator;
+  auto query = queryGenerator.generateGetAccountAssetsTransactionsWithPager(
+      0, "123", 0, "test", {"a", "b"}, Pager{iroha::hash256_t{}, 1});
+  auto pb_query = queryFactory.serialize(query);
+  ASSERT_TRUE(pb_query.has_value());
+  auto res_query = queryFactory.deserialize(pb_query.value());
+  ASSERT_TRUE(res_query.has_value());
+  // TODO: overload operator == for queries and replace with it
```

Comment:
Pls, add a task in Jira about adding operator== for queries.

Reply:
It seems: https://soramitsu.atlassian.net/browse/IR-512